src/Core/Content/Product/Aggregate/ProductSearchConfig/ProductSearchConfigHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Product\Aggregate\ProductSearchConfig;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('inventory')]
+class ProductSearchConfigHelper
```

Comment:
Not sure if this is the perfect place, but a helper is definitely needed with the amount of times it is needed